src/components/interactive-map.js

```diff
@@ -4,18 +4,61 @@ import autobind from '../utils/autobind';
 
 import StaticMap from './static-map';
 import MapControls from './map-controls';
-import MapState from '../utils/map-state';
+import MapState, {MAPBOX_MAX_PITCH, MAPBOX_MAX_ZOOM} from '../utils/map-state';
 import {PerspectiveMercatorViewport} from 'viewport-mercator-project';
 
 const propTypes = Object.assign({}, StaticMap.propTypes, {
-  displayConstraints: PropTypes.object.isRequired
+  // Additional props on top of StaticMap
+
+  /** Viewport constraints */
+  // Max zoom level
+  maxZoom: PropTypes.number,
+  // Min zoom level
+  minZoom: PropTypes.number,
+  // Max pitch in degrees
+  maxPitch: PropTypes.number,
+  // Min pitch in degrees
+  minPitch: PropTypes.number,
+
```

Comment:
Technically I believe the system supports constraints on all props. No value in listing them here, but add a comment to this effect.